hack/kind_install_for_capd.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
```

Comment:
Please add the copyright header to this script:

https://github.com/kubernetes-sigs/cluster-api/blob/master/hack/ensure-kubectl.sh#L3-L15

Reply:
I'm wondering about this as this is mostly taken from the KIND script - is it appropriate to add a header to it?

Reply:
Yes